requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+coverage
+pytest
+pytest-cov
+requests-mock
```

Comment:
leaving these unversioned for now - an upcoming PR will switch to using a more up-to-date python, and I will add the versions then.